### PR TITLE
Cleanup repository a bit

### PR DESCRIPTION
**Pull request by egonelbre**
1. Move tools from `cmd` to `tools`
2. Remove some old code
3. Move code to examples

### CODE REVIEW

File: cmd/pdbinspect/main.go [hunk 1]
```diff
@@ -1,211 +0,0 @@
-// Copyright (C) 2019 Storj Labs, Inc.
```
Comment (mobyvb) [inline cmd/pdbinspect/main.go]: Why is pdbinspect being removed? Is its functionality somewhere else?

Comment (egonelbre) [reply cmd/pdbinspect/main.go]: This seems outdated and will get outdated with more time. Better not to worry about this when refactoring.

File: pkg/process/metrics.go [hunk 2]
```diff
@@ -28,8 +28,7 @@ var (
 
 // InitMetrics initializes telemetry reporting. Makes a telemetry.Client and calls
 // its Run() method in a goroutine.
-func InitMetrics(ctx context.Context, r *monkit.Registry, instanceID string) (
-	err error) {
+func InitMetrics(ctx context.Context, r *monkit.Registry, instanceID string) (err error) {
```
[the remark this answers is not in the document]
Comment (egonelbre) [reply pkg/process/metrics.go]: I just made it single line no other changes.

**Comment by egonelbre** [comment]
@jtolds I'll revert metric-receiver and grpc-debug.

**Pull request merged**